

def internal_server_error():
    return 'ERROR', 500
